doc/source/api.rst

```diff
@@ -111,9 +111,7 @@ Google BigQuery
    read_gbq
    to_gbq
    generate_bq_schema
```

Comment:
Since we are making changes to `generate_bq_schema`, I was thinking 
 `generate_bq_schema` could be moved to a private function and removed from the docs. I'm not sure it will be used by many users and would simplify the api.